server/controllers/events/events_controller.go

```diff
@@ -306,9 +306,18 @@ func (e *VCSEventsController) HandleGithubCommentEvent(event *github.IssueCommen
 		}
 	}
 
+	body := event.GetComment().GetBody()
+
+	if strings.HasPrefix(body, "atlantis ") {
+		err = e.VCSClient.ReactToComment(baseRepo, *event.Comment.ID, "eyes")
```

Comment:
Thank you for the pr.

Please make the emojis configurable from the server config and overwritable in the repo config

Reply:
Nice feature. The rapid feedback improves the developer experience. I also want to add 🚀 .

Reply:
It might be better to move react logic in here.

https://github.com/runatlantis/atlantis/blob/9a38e354936fbf93282e4e934fa3439fc3bfd102/server/controllers/events/events_controller.go#L574

Reply:
@krrrr38 I would have preferred that as well, but there you don't have access to the event object anymore so I don't see a way to get to the comment id.